3.train_ebimage/1.random_forest/results/all_scores.csv

```diff
@@ -0,0 +1,9 @@
+,score
```

Comment:
this is ok for now, but generally good [tidy data practice](https://garrettgman.github.io/tidying/) is to ensure that all columns in a dataframe have a column name.

For this particular case, it looks like `all_scores.csv` was output from a `pd.to_csv()` call with the default `index = True`. Can switch to `index = False` to remove the first column.

This doesn't impact results in anyway, so feel free to ignore this comment!

Reply:
Good point. I will change it for the newly generated csv files